frame/collective/src/lib.rs

```diff
@@ -907,6 +929,9 @@ impl<T: Config<I>, I: 'static> ChangeMembers<T::AccountId> for Pallet<T, I> {
 				T::MaxMembers::get(),
 			);
 		}
+
+		let bounded_new = BoundedVec::<T::AccountId, T::MaxMembers>::truncating_from(new.to_vec());
```

Comment:
can this be done better? for example without calling `to_vec`?

Reply:
Since `new` is a slice here, I think it's best that you instead create a `truncating_from` implementation on `BoundedSlice` and create one instance here. Since `BoundedSlice` encodes like `BoundedVec`, we can still put it into the `Members` storage item without the compiler complaining.

Reply:
I took the liberty to move this method from `BoundedVec` to `BoundedSlice`, and made this variable into a `BoundedSlice` instead in 47c36b88e239304bd8f1d3cb69794bdb15765a16.